status-im/status-react
# Button Refactors Final Final

fixes: https://github.com/status-im/status-mobile/issues/16535

This pr updates the use of the button component.
I renamed a lot of props to align better with Figma and best practices
i.e 
**prop names**
`icon` -> `icon-only?`
`before` -> `icon-left`
`after` -> `icon-right`
`above` -> `icon-top`
`style` -> `container-style`
`disabled` -> `disabled?`

removed all uses of combined types, e.g `blur-bg` -> `type :grey` & `:background :blur`

also I removed some internal styles as they were duplicates an not needed.

Updated all uses across the codebase to now use this api

Also fixed an alignment issue with the outline button variant where it was not sitting correctly 

To Test: Each and every button 

Additionally I found a number of uses of customization color for in the application that were not set correctly, this meant on first login the user's color was not showing. Now it is using the right `sub` for data and should show 👍
